utils/swift_build_support/swift_build_support/targets.py

```diff
@@ -68,6 +68,17 @@ def contains(self, target_name):
                 return True
         return False
 
+    def swift_flags(self, args):
+        """
+        Swift compiler flags for a platform, useful for cross-compiling
+        """
+        return ''
+
+    def cmake_options(self, args):
+        """
+        CMake flags to build for a platform, useful for cross-compiling
+        """
+        return ''
```

Comment:
One question that I have (and I am fine with you doing this in a subsequent commit) is if it would be better to error and force people to specify this. That being said, I don't feel too wedded to this.

Reply:
Hmm, the problem is that wouldn't that break the current cross-compilation of the Apple Silicon toolchain on the macOS x86_64 CI? I tried to keep this as non-intrusive as possible, because I don't want to interfere with other use cases.